cmd/wallet-web/src/components/ToastNotification/ToastNotification.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div v-show="close" :class="classname">
+    <div class="xs:flex-none px-4">
+      <img class="sm:w-10 h-6 sm:h-8 w-7" :src="icon" />
+    </div>
+    <div class="flex-col">
+      <span class="text-sm xs:text-lg font-bold leading-relaxed align-left">{{ title }}</span>
+      <div>
+        <span class="text-xs xs:text-sm leading-6 align-left">{{ description }}</span>
+      </div>
+    </div>
+    <div class="xs:flex-none px-4 align-middle">
+      <button type="button" @click="close = !close">
+        <img class="w-6 h-6" src="@/assets/img/close-icon.svg" />
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'ToastNotification',
+  props: {
```

Comment:
can you make configurable too?
and make `@/assets/img/triangle-exclamation.svg` as default.

Reply:
i didnt make confiugurable as each toast will have its own icon, like success and warning would not use ! sign icon. This can be added when we register. If we want i can make it

Reply:
> i didnt make confiugurable as each toast will have its own icon, like success and warning would not use ! sign icon. This can be added when we register. If we want i can make it

I think this makes sense the way we have it now